### PR TITLE
samples: Bluetooth: direction_finding_connectionless_rx: Add config. and DTS overlay to allow build for nRF52820

### PR DESCRIPTION
**Pull request by ppryga-nordic**
Add configuration and DTS overlay for nRF52830DK_nrf52820 boards.
Required to enable support for nRF52820 SOC.

This PR should be merged after: #36159

### CODE REVIEW

File: samples/bluetooth/direction_finding_connectionless_rx/README.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ periodic advertising PDUs.
 Requirements
 ************
 
+* nRF52833DK board with nRF52833 SOC
 * nRF52833DK board with nRF52833 SOC
```
Comment (Thalley) [inline samples/bluetooth/direction_finding_connectionless_rx/README.rst:17]: Duplicated line

File: samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52820.overlay [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+&radio {
+	status = "okay";
+	/* This is an example number of antennas that may be available
+	 * on antenna matrix board.
+	 */
+	dfe-antenna-num = < 10 >;
+	/* This is an example switch pattern that will be used to set an
+	 * antenna for Tx PDU (period before start of Tx CTE).
+	 */
+	dfe-pdu-antenna = <0x1>;
```
Comment (Thalley) [inline samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52820.overlay:16]: One place you have `<value>` and the other `< value >` - I guess spacing should be consistent.

**Review by Thalley — CHANGES_REQUESTED** [review]

File: samples/bluetooth/direction_finding_connectionless_rx/README.rst (hunk 1 above)
Comment (Thalley) [reply samples/bluetooth/direction_finding_connectionless_rx/README.rst:17]: I guess one of them should be with nRF52820 SOC?

Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_rx/README.rst]: Yes, it should be nRF52820, thank you. It is corrected.

File: samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52820.overlay (hunk 2 above)
Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52820.overlay]: Removed unnecessary spaces.

**Review by Thalley — APPROVED** [review]

**Comment by ppryga** [comment]
@jhedberg @joerchan could you take a look on the PR?

**Review by jhedberg — APPROVED** [review]

**Pull request merged**